resources/js/controllers/fields/input_controller.js

```diff
@@ -14,7 +14,8 @@ export default class extends Controller {
         } catch (e) {
             // default
         }
-
-        Inputmask(mask).mask(element);
+        if (mask != '') {
```

Comment:
As far as I remember, the empty string returns `false`.
```php
if (mask) {
    Inputmask(mask).mask(element);
}
```

Reply:
ok, thanks.